### ISSUE TITLE
Dynamic Playlist Issues

### ISSUE THREAD

**Issue by emilysnowflake** [ArtisTechMedia/ccMixterHost#11]
Dynamic playlists will only display 15 tracks, even when 25 or 50 are selected and submitted.  For example: http://ccmixter.org/playlist/browse/18021

There used to be an option to display 100 tracks, that is missing (but wouldn't work anyway at the moment).

These are important for the upcoming remix event. 

Note: the sorting by date, name, rating and artist does seem to be working.

**Comment by emilysnowflake** [comment]
It looks like the sorting and display is working up to 50 items. It used to be 100 items. It would be great if it would display up to 100 again.....

**Comment by emilysnowflake** [comment]
Any chance we could 100 items, or a paginated display, before we start the project Tuesday? I'm certain we'll have more than 50 items in each dynamic playlist (pells, samples, remixes) pretty quickly. Other thoughts? Almost all past projects have had 100 each. It is nice to have them all available in playlists that can be embedded on other sites (like the U of U site).
